src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -44,35 +45,70 @@
     private String host = "slack.com";
     private String baseUrl;
     private String teamDomain;
-    private String token;
-    private String authTokenCredentialId;
+    private String token = null;
+    private String authTokenCredentialId = null;
     private boolean botUser;
     private String[] roomIds;
     private boolean replyBroadcast;
     private String responseString = null;
+    private String populatedToken = null;
 
+    /**
+     * @deprecated use {@link #StandardSlackService(String, String, boolean, String, boolean, String)} instead}
+     */
+    @Deprecated
     public StandardSlackService(String baseUrl, String teamDomain, String authTokenCredentialId, boolean botUser, String roomId) {
         this(baseUrl, teamDomain, null, authTokenCredentialId, botUser, roomId, false);
     }
 
+    /**
+     * @deprecated use {@link #StandardSlackService(String, String, boolean, String, boolean, String)} instead}
+     */
+    @Deprecated
```

Comment:
I'll check this when I test the code out, but can you make sure that none of these constructors are called anymore internally to the codebase?

Reply:
just had a look, `StandardSlackServiceTest` are using the old constructors, could you update that please?
and any other places

Reply:
They're only used from tests.